docs/protocol/core/v3.0.rst

```diff
@@ -1194,6 +1196,17 @@ A **writeable store** supports the following operations:
     | Parameter: `prefix`
     | Output: none
 
+.. note::
+
+   Some KV stores do allows creations and update of keys, but not deletion.
+   For example Zip archive do not allow removal of content without recreating
+   the full archive.
+
+   Unability to delete can affect ability to rename keys as well, as a ranema as
+   often a succession or an atomic combinaison of deleting data and setting it
+   to a new name.
```

Comment:
Minor readability suggestions/ typo fixes:


> Some KV stores do allow creation and update of keys, but not deletion. For example, Zip archives do not allow removal of content without recreating the full archive.
>
> Inability to delete can affect ability to rename keys as well, as a renames are often a sequence or atomic combination of deletion and creation.

Reply:
@grlee77: do you have a parallel branch for @clbarnes' suggestion?